root/programs/C#/Frameworks/Infrastructure/Public/IO/SymmetricCryptography.cs

```diff
@@ -432,13 +438,15 @@ private static SymmetricAlgorithm CreateSymmetricAlgorithm(EnumSymmetricAlgorith
             SymmetricAlgorithm sa = null;
 
             // AesCryptoServiceProvider, AesManagedは3.5からの提供。
-
-            if (esa == EnumSymmetricAlgorithm.AesCryptoServiceProvider)
-            {
-                // AesCryptoServiceProviderサービスプロバイダ
-                sa = AesCryptoServiceProvider.Create(); // devps(1703)
-            }
-            else if (esa == EnumSymmetricAlgorithm.AesManaged)
+            // ポリシーに合わせて、AesCryptoServiceProviderを削除。
```

Comment:
@yusukemaegawa 

Cryptographic provider selection policy.
- Managed library is top priority.
- Then CSP library. Finally CNG library.
- There is a possibility that the priority of CSP and CNG will change in the future.

I will add an explanation about this.